scranShipper/src/Display/RestaurantDisplay.jsx

```diff
@@ -0,0 +1,16 @@
+function RestaurantDisplay(props){
+    const makeRestaurant = (restaurant) => {
+        return (<>
+        <button> {restaurant.name} </button>
+        </>)
+    }
+
+    return (
+        <>
+            {props.restaurantList.map(makeRestaurant)
+            }
```

Comment:
Tidy this up by moving the } to the end of the line instead of having it on another line